<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphVentana">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>801</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>801</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Resumen de cuenta mensual</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>781</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtConsumo"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblConsumo">
        <property name="text">
         <string>Consumo (KW/mes)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblCosto">
        <property name="text">
         <string>Costo (Q/mes)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="txtCosto"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTipo">
        <property name="text">
         <string>Tipo de cuenta</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtTipo"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnResultados">
        <property name="text">
         <string>Mostrar resultados</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>801</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Informaci√≥n adicional</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>781</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtLongitud"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblLongitud">
        <property name="text">
         <string>Longitud de cable (m)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="lblCalibre">
        <property name="text">
         <string>Calibre a utilizar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLineEdit" name="txtCalibre"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnCalcular">
        <property name="text">
         <string>Calcular</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionAgregar"/>
    <addaction name="actionGestionar"/>
   </widget>
   <addaction name="menuEditar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReiniciar">
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionAgregar_electrodom_stico">
   <property name="text">
    <string>Agregar nuevo</string>
   </property>
  </action>
  <action name="actionAgregar_nuevo">
   <property name="text">
    <string>Agregar</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="actionReiniciar_2">
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </action>
  <action name="actionAgregar">
   <property name="text">
    <string>Agregar</string>
   </property>
  </action>
  <action name="actionGestionar">
   <property name="text">
    <string>Gestionar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
